spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/security/LoginController.java

```diff
@@ -49,13 +63,4 @@ public String authorize(
 		return session.getId();
 	}
 
-	@RequestMapping("/admin-ui/")
```

Comment:
Hangover from Spring XD days ... we now have `/dashboard` handled in class `UiController`